wordpress-mobile/WordPress-Android
# Issue/7138 add email signup

### Fix
Add email signup screen as described in https://github.com/wordpress-mobile/WordPress-Android/issues/7138.  The new [`SignupEmailFragment` class](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/7138-add-email-signup/WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEmailFragment.java) is similar to the [`LoginEmailFragment` class](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/7138-add-email-signup/libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java), but not entirely.  We can create a `BaseEmailFragment` class to help abstract common code as an optimization when we move the signup code into the `WordPressLoginFlow` library.  I chose not to do that at this time to avoid conflicts with ongoing development.  The "Continue to magic link" message denotes the flow that will be handled in the subsequent magic link issue.

### Test
0. Clear app data.
1. Tap ***Sign Up*** button.
2. Tap ***Sign Up with Email*** button.
3. Enter email address associated with WordPress.com account.
4. Notice message and login flow.
5. Tap back arrow.
6. Enter email address not associated with WordPress.com account.
7. Notice "Continue to magic link" message.
